cfme/tests/cloud/test_providers.py

```diff
@@ -1250,8 +1250,8 @@ def test_ec2_create_sns_topic():
 
 
 @test_requirements.ec2
-@pytest.mark.manual
-def test_ec2_add_delete_add_provider():
+@pytest.mark.provider([EC2Provider], scope="function", override=True, selector=ONE)
```

Comment:
override kwarg is no longer necessary, and is essentially ignored (generates a log message)